build.gradle

```diff
@@ -1,14 +1,20 @@
+buildscript {
+    dependencies {
+        classpath "gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.0.0"
```

Comment:
suggest using plugins block instead: https://plugins.gradle.org/plugin/com.github.spotbugs

_unless_ you are specifically trying to keep the PRs for upgrading gradle and upgrading spotbugs separate.